### ISSUE TITLE
el9 build failure due to missing packages

### ISSUE THREAD

**Issue by sandrobonazzola** [oVirt/ovirt-node-ng-image#22]
**Describe the bug**
Building for el9 fails with:
```
16:56:00,737 INFO anaconda:program: No match for argument: ovirt-release-host-node
16:56:00,737 INFO anaconda:program: No match for argument: ovirt-node-ng-image-update-placeholder
16:56:00,737 INFO anaconda:program: Error: Unable to find a match: ovirt-release-host-node ovirt-node-ng-image-update-placeholder
```

**Comment by sandrobonazzola** [comment]
This seems to be caused by missing repos:
```
16:56:00,737 INFO anaconda:program: CentOS Stream 9 - BaseOS                        495 kB/s | 5.6 MB     00:11
16:56:00,737 INFO anaconda:program: CentOS Stream 9 - AppStream                     9.1 MB/s |  12 MB     00:01
16:56:00,737 INFO anaconda:program: CentOS Stream 9 - CRB                           437 kB/s | 3.9 MB     00:09
16:56:00,737 INFO anaconda:program: CentOS Stream 9 - Extras packages                37 kB/s | 6.3 kB     00:00
```

**Comment by sandrobonazzola** [comment]
Missing repos due to openssl changes causing:
```
16:56:00,729 INFO anaconda:program: Problem opening package ovirt-release-master-4.5.0-0.0.master.20220302132934.gitdc45bfd.el9.noarch.rpm
16:56:00,729 INFO anaconda:program: The downloaded packages were saved in cache until the next successful transaction.
16:56:00,729 INFO anaconda:program: You can remove cached packages by executing 'dnf clean packages'.
16:56:00,729 INFO anaconda:program: Error: GPG check FAILED
```

**Comment by sandrobonazzola** [comment]
fixed via #25
